fincode_api_client/openapi_client.gemspec

```
@@ -0,0 +1,39 @@
+Gem::Specification.new do |s|
+  s.name        = "openapi_client"
+  s.version     = OpenapiClient::VERSION
+  s.platform    = Gem::Platform::RUBY
+  s.authors     = ["OpenAPI-Generator"]
+  s.email       = [""]
+  s.homepage    = "https://openapi-generator.tech"
+  s.summary     = "fincode API Ruby Gem"
+  s.description = "fincodeのAPIリファレンスです。\\ fincodeは決済情報のほか様々なリソースを管理できるため、リソース指向なREST APIをベースとしたインターフェイス設計のもとでAPIを提供しています。\\ APIの呼び出しにはAPIキーによる認証が必要であり、リクエストの際に`Authorization`ヘッダーに有効なAPIキーをセットすることで認証できます。 "
+  s.license     = "Unlicense"
+  s.required_ruby_version = ">= 2.7"
+  s.metadata    = {}
```

RuboCop (1.84.0)

[convention] Operator `=` should be surrounded by a single space.

(Layout/SpaceAroundOperators)